src/bosh-director/lib/bosh/director/jobs/update_stemcell.rb

```diff
@@ -141,6 +151,20 @@ def update_steps
         steps += 1 if @stemcell_sha1 # also verify remote stemcell
         steps + cloud_factory.all_configured_clouds.count * 3 # check, upload and save for each cloud
       end
+
+      def is_supported?(cloud)
+        info = cloud[:cpi].info
+        unless @stemcell_formats.nil? || info["stemcell_formats"].nil?
+          return (info["stemcell_formats"] & @stemcell_formats).empty? ? false : true
+        else
+          logger.info("There is no enough information to check if stemcell format is supported")
+          return true
+        end
+      rescue
```

Comment:
by default with old versions of bosh_cpi, cpis return another error - InvalidCall, converted to UnknownError with varied text messages. Is it particular enough to check?